authentication_authorization/README.md

```diff
@@ -126,17 +126,18 @@ Let's add some sugar to our templates while we're at it. First we will add some
 ```django
 <div class="page-header">
     {% if user.is_authenticated %}
+        <a href="{% url 'logout' %}" class="top-menu"><span class="glyphicon glyphicon-off"></span></a>
```

Comment:
I think placing the "logout" button besides a button to create a new blogpost will result in accidental clicking of logout. I will not vote on that solution, sorry :(